tests/logictest/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM python:3.10.4-alpine3.15
 
 RUN /usr/local/bin/python -m pip install --upgrade pip && pip install --user mysql-connector six PyHamcrest requests environs fire
-RUN pip install https://github.com/youngsofun/clickhouse-sqlalchemy/archive/refs/heads/databend.zip
+RUN pip install https://github.com/youngsofun/clickhouse-sqlalchemy/archive/dfd9ba3540dcb3c88f0850ef30746738bc6915f7.zip
```

Comment:
Oh, that's my fault. The latest commit on databend branch is `a116e3162c699c12e63a689385b547f639c13018`

https://github.com/youngsofun/clickhouse-sqlalchemy/commits/databend